pkg/syncer/pvcsi_fullsync.go

```diff
@@ -47,6 +53,8 @@ func PvcsiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) er
 			(time.Since(fullSyncStartTime)).Seconds())
 	}()
 
+	AddNodeAffinityRulesOnPV(ctx, metadataSyncer)
```

Comment:
Can we check for WorkloadDomainIsolationSupported FSS here? We don't want this to run when SV is not upgraded as it is of no use.

Reply:
done